charts/opensrp-web/values.yaml

```diff
@@ -186,6 +187,21 @@ express:
   sessionPath: "/"
   reactBuildPath: "/usr/src/web"
   nodeEnv: "production"
+  csp:
+    report-uri: "https://{{ .Values.sentry.domain }}/api/{{ .Values.sentry.projectId }}/security/?sentry_key={{ .Values.sentry.sentryKey }}&sentry_environment={{ .Values.sentry.environment }}&sentry_release={{ .Release.Name }}"
+    default-src:
+      - "'self'"
+      - "'unsafe-inline'"
```

Comment:
recommend removing `unsafe-inline` or even the whole default-src directive in favor of specific directives. This is to avoid unintended consequences like giving wrong whitelist permission.